torchmultimodal/utils/common.py

```diff
@@ -127,20 +127,20 @@ def load_model(
 
 
 class ModelOutput(OrderedDict):
-    def keys(self):
+    def keys(self) -> Any:
```

Comment:
Should we use `Generator` as the type hint for `yield`? 

https://docs.python.org/3/library/typing.html#typing.Generator

Reply:
Ultimately did not adopt this suggestion as ModelOutput inherits from OrderedDict which has its own type for this. [Ref](https://gist.github.com/joequery/12332f410a05e6c7c949#file-ordered_dict-py-L175)

Reply:
The generator is higher than the keys right? I.e., we are returning a generator instead of the keys themselves.

Reply:
I've tried setting the return type to Generator but mypy gets quite angry:
```
torchmultimodal/utils/common.py:168:5: error: Return type "Generator[Any, Any, Any]" of "keys" incompatible with return type
"_OrderedDictKeysView[Any, Any]" in supertype "OrderedDict"  [override]
        def keys(self) -> Generator:
        ^
torchmultimodal/utils/common.py:168:5: error: Return type "Generator[Any, Any, Any]" of "keys" incompatible with return type
"dict_keys[Any, Any]" in supertype "dict"  [override]
        def keys(self) -> Generator:
        ^
torchmultimodal/utils/common.py:168:5: error: Return type "Generator[Any, Any, Any]" of "keys" incompatible with return type
"KeysView[Any]" in supertype "Mapping"  [override]
        def keys(self) -> Generator:
```
I think setting it to Any is the simplest although not ideal solution